src/patternfly/components/AppLauncher/examples/AppLauncher.md

```diff
@@ -197,6 +197,112 @@ import './AppLauncher.css'
 {{/app-launcher}}
 ```
 
+```hbs title=Favorites
+{{#> app-launcher id="app-launcher-sections-dividers-icons-and-external-links" app-launcher--IsExpanded="true" app-launcher--IsGrouped="true"}}
+  {{#> app-launcher-menu}}
+    {{#> app-launcher-menu-input}}
+      {{#> form-control controlType="input" input="true" form-control--attribute=(concat 'type="search" placeholder="Filter by name..." id="' id '-text-input" name="textInput1"')}}
+        {{/form-control}}
+    {{/app-launcher-menu-input}}
+    {{#> app-launcher-group}}
+      {{#> app-launcher-group-title}}
+        Favorites
+      {{/app-launcher-group-title}}
+      <ul>
```

Comment:
Same for this? Should this be a partial?

Reply:
Since it's not a BEM element, I'm not really concerned with it being in a partial. Same way for the dropdown and options menu group lists.

Reply:
👍